yt_dlp/extractor/facebook.py

```diff
@@ -478,9 +480,35 @@ def _perform_login(self, username, password):
             return
 
     def _extract_from_url(self, url, video_id):
+        cookies = self._get_cookies(url)
+        # user passed logged-in cookies or attempted to login
+        login_data = cookies.get('c_user') and cookies.get('xs')
+        logged_in = False
+
         webpage = self._download_webpage(
             url.replace('://m.facebook.com/', '://www.facebook.com/'), video_id)
 
+        sjs_data = [self._parse_json(j, video_id, fatal=False) for j in re.findall(
+            r'data-sjs>({.*?ScheduledServerJS.*?})</script>', webpage)]
```

Comment:
this is duplicating the `post_data` var from inside the `_extract_metadata` method